TOMP-API.yaml

```diff
@@ -2594,6 +2594,15 @@ components:
         type: string
       description: The ID of the sending maas operator
       example: 1324A-DFB3482-32ACD
+    addressedTo:
+      in: header
+      name: addressed-to
+      required: false
+      schema:
+        type: string
+      description: The ID of the maas operator that has to receive this message
```

Comment:
Should we add some explanations what this ID is? For instance, in the case of Nazza, in what field of the planning-options response do the fill in the id of the sub TO so that a MP can use that ID in this header? Could this header also be used to simply put the name of the (sub) TO?